content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -56,8 +60,8 @@ To view or edit the current configuration, view or edit the contents of `/etc/cr
 
 ```shell
 cat /etc/crictl.yaml
-runtime-endpoint: unix:///var/run/dockershim.sock
-image-endpoint: unix:///var/run/dockershim.sock
+runtime-endpoint: unix:///var/run/containerd.sock
```

Comment:
this is wrong, containerd socket lives in `/run/containerd/containerd.sock`

Reply:
Led into this with a "for example, the configuration for the containerd container runtime would be similar to this:"